docs/Guides/managing-your-organization/managing-accounts.md

```diff
@@ -42,7 +42,7 @@ Type a name for the account and choose a region. You won't be able to change the
 Then you will see your new account on the **Accounts management** page. 
 
 {: .note}
-There can be up to twenty accounts per organization. If you need to create more, please contact the Firebolt support team.
+There can be up to 25 accounts per organization. If you have a need for additional account creations beyond this limit, contact [Firebolt Support](https://docs.firebolt.io/godocs/Reference/help-menu.html) for assistance. Our team can provide guidance and, if appropriate, adjust your account settings to accommodate your needs.
```

Comment:
With current implementation, 25 is the monthly limit for CREATE ACCOUNT, but max. numbers of accounts is still 20.

Reply:
Thanks for the catch @matino. I aligned the language to 20 accounts and a limit of CREATE ACCOUNT of 25. This hits the reqs for a 5 attempt buffer.

Reply:
LGTM, unfortunately I can't approve it nor merge it. @krishfirebolt @arushi-firebolt can you help us plz?